README.md

```diff
@@ -1,164 +1,222 @@
-# Tower CLI app
+Nextflow Tower CLI
+==================
 
-Proof of concept of Tower command line client app based on Java native compilation, Micronaut HTTP declarative client and Tower domain classes
+`tw` is Nextflow Tower on the command line. It brings Tower concepts including Pipelines, Actions and Compute Environments to the terminal.
 
-## Installation
+![tw](/tw-screenshot.png)
```

Comment:
Can we create this image relative to the `rnaseq` pipeline and not `nanoseq`? Be good to be consistent in terms of which pipeline we use as our "gold standard"?